### ISSUE TITLE
Stuck at Initializing environment

### ISSUE THREAD

**Issue by vorburger** [editorconfig-checker/editorconfig-checker.python#24]
I've added `editorconfig-checker.python` to a project (private for now, but I can probably share it next week, if you want to debug it further) and it seems to just be "stuck" on _Initializing environment_ more or less forever (I've waited at least a few minutes).

Perhaps it's missing some pre-requisites or something, but it would be nice if it would say where/why it gets stuck.

```
$ pre-commit run --verbose
[INFO] Initializing environment for https://github.com/editorconfig-checker/editorconfig-checker.python.
```

**Comment by vorburger** [comment]
```
$ pre-commit run --verbose
[INFO] Initializing environment for https://github.com/editorconfig-checker/editorconfig-checker.python.

[... AFTER WHAT TO ME SEEMED LIKE A VERY VERY LONG TIME...]

An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'checkout', 'v2.7.1')
return code: 1
stdout: (none)
stderr:
    error: pathspec 'v2.7.1' did not match any file(s) known to git
Check the log at /home/vorburger/.cache/pre-commit/pre-commit.log
```

I had used `rev: v2.7.1` instead of `rev: 2.7.1` - after changing that, it works now.

Closing; apologies for the noise.

**Comment by mmicu** [comment]
No worries. Thanks for posting the solution as well.
